lib/create-contentful-api.js

```diff
@@ -198,9 +195,22 @@ export default function createContentfulApi ({
     const resolveLinks = shouldLinksResolve(query)
     return pagedSync(http, query, resolveLinks)
   }
+  /**
+   * Parse a Given raw json data coming from the collection endpoint
+   * into collection of entry objects.
+   * Links will be resolved also
+   * @param {Object} data json data represting collection of entries
+   * @example
+   * let parsedData = client.parseEntries(data)
+   * console.log(parsedData)
```

Comment:
can we make it more clear what the power of this is?

it would be nice to see in the example that linking takes place. So for me to really understand it seeing an example json object and then calling a linked property would be awesome.

```
let data = {};

console.log( data.entries[0].fields.foo ); // undefined

let parsedData = client.parseEntries(data);

console.log( data.entries[0].fields.foo ); // foo
```

what do you think?